src/postgres/src/test/regress/yb_pg_groupingsets_schedule

```diff
@@ -0,0 +1,15 @@
+# src/test/regress/yb_pg_groupingsets_schedule
+#
+####################################################################################################
+# This includes tests for Group
+####################################################################################################
+# Dependencies for `yb_pg_groupingsets`
+test: yb_pg_int8
+
+# Chain of dependencies for `yb_pg_window`
```

Comment:
Looks like a copy/paste mistake.  Can you make sure that `yb_pg_copy` is really needed for `yb_pg_groupingsets`?

Reply:
See: https://github.com/yugabyte/yugabyte-db/pull/5190#discussion_r462827939